transifex/native/daemon.py

```diff
@@ -13,14 +13,16 @@ class DaemonicThread(threading.Thread):
     daemon = True
     should_exit = False
 
-    def start_daemon(self, interval=10):
+    def start_daemon(self, interval=None):
```

Comment:
I think you should use the INTERVAL setting here, and if it's not set, default back to `10*60`.